Project.toml

```diff
@@ -4,6 +4,7 @@ version = "0.6.1"
 
 [deps]
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
```

Comment:
I'm a bit puzzled by this. Didn't we already depend on Conda?

Reply:
I'll switch that extras

Reply:
I just want to understand what's going on, it's been a while. When a new user `]add ScikitLearn`, it should generally work. From some investigation, it [seems](https://github.com/ValdarT/ScikitLearn.jl/blob/5bbb33fc58e9e7b5199be962d47cec2b0b987bb3/REQUIRE) that `Conda` was in `REQUIRE`. I guess when we created Project.toml, we forgot to put it in there. So I think this PR is correct. `Conda` should be in `Project.toml`. No?

Reply:
Installing via PyCall seemed better though. I hate to have to force the user to user to use Conda by default. They may already have a `sklearn` installed on a different python and may want to use it.

Reply:
`@sk_import` calls `import_sklearn`, which calls `PyCall.pyimport_conda`, which first tries to import the module

```julia
 try
        pyimport(modulename)
```

If the user already has a working `scikit-learn` installed (accessible via PyCall), it should be used, and `Conda` won't install anything. I do believe that's for the best. A lot of `ScikitLearn.jl` users are relatively new to julia, and making their lives easier with good defaults is important.